src/lib.rs

```diff
@@ -1802,6 +1802,10 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![warn(missing_docs)]
 
+// Needed for linking
+#[cfg(all(feature = "rb-sys-interop", feature = "embed"))]
+extern crate rb_sys as __rb_sys_interop;
```

Comment:
I don't understand why this is needed. Can you point me to some docs or do you have an example I can play with to get an understanding? Or even better a test that fails without it?

Reply:
I don't think this should be necessary, but for some reason had to do it while static linking. Let me test again against a static Ruby.